### PR TITLE
PR: Conda env

### PR DESCRIPTION
**Pull request by martindurant**
Required for importing .yaml-specified environments.

### CODE REVIEW

File: conda_manager/api/conda_api.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 import re
 import sys
 import yaml
+import time
```
Comment (goanpeca) [inline conda_manager/api/conda_api.py:17]: could you aranga alphabetically?

Comment (ccordoba12) [inline conda_manager/api/conda_api.py:17]: This import seems unnecessary.

File: conda_manager/api/conda_api.py [hunk 2]
```diff
@@ -107,10 +108,10 @@ def __init__(self, cmd_list, parse=False, pip=False, callback=None,
         self._timer = QTimer()
         self._process = QProcess()
 
-        self._timer.setInterval(50)
+        self._timer.setInterval(150)
 
         self._timer.timeout.connect(self._communicate)
-        self._process.finished.connect(self._communicate)
+        # self._process.finished.connect(self._communicate)
```
Comment (Nodd) [inline conda_manager/api/conda_api.py:114]: If this line is not needed anymore, it should be removed instead of commented out.

**Comment by Nodd** [comment]
You merged before fixing the comments ?

**Comment by goanpeca** [comment]
I already fix them in a separate commit after this merge

**Pull request merged**